src/utils/toast.js

```diff
@@ -0,0 +1,18 @@
+const makeToast = (message, type) => {
```

Comment:
should just be called `toast`

Reply:
![image](https://github.com/acm-ucr/hackathon-website/assets/120443522/057f93b8-2742-4fef-b73b-cc4c31637660)
I am planning to just call it toaster instead of Toaster because of this error from ESLint. On it right now